internal/logger/logger.go

```diff
@@ -114,10 +104,12 @@ func NewLogger(level string, key *string, secret *string, region *string, group
 	}
 
 	if key != nil {
-		log.SetFormatter(NewCloudwatchFormatter())
```

Comment:
Why was this line removed?

Reply:
it moved to line 108. Just assigning the formatter to a variable since it's used on line 112 as well.